*** tests/A-007-002-M-ok.out	2015-11-26 20:23:19.222358235 +0000
--- tests/A-007-002-M-ok.outhyp	2015-11-27 11:48:01.617223879 +0000
***************
*** 99,105 ****
  Escolha uma opção: Título: 
  Secções de topo: 0
  Dimensão do documento (bytes): 18
! Identificadores únicos: 2
  Menu Principal
  1 - Novo
  2 - Abrir
--- 99,105 ----
  Escolha uma opção: Título: 
  Secções de topo: 0
  Dimensão do documento (bytes): 18
! Identificadores únicos: 0
  Menu Principal
  1 - Novo
  2 - Abrir
