*** tests/A-005-004-M-ok.out	2015-11-24 08:56:06.000000000 +0000
--- tests/A-005-004-M-ok.outhyp	2015-11-25 13:49:16.054821383 +0000
***************
*** 18,27 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Título: Objects in Space
! Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
! Secções de topo: 3
! Dimensão do documento (bytes): 347
  Identificadores únicos: 0
  Menu Principal
  1 - Novo
--- 18,26 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Título: 
! Secções de topo: 0
! Dimensão do documento (bytes): 0
  Identificadores únicos: 0
  Menu Principal
  1 - Novo
***************
*** 33,42 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Título: Objects in Space
! Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
! Secções de topo: 3
! Dimensão do documento (bytes): 347
  Identificadores únicos: 0
  Menu Principal
  1 - Novo
--- 32,40 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Título: 
! Secções de topo: 0
! Dimensão do documento (bytes): 0
  Identificadores únicos: 0
  Menu Principal
  1 - Novo
