*** tests/A-009-002-M-ok.out	2015-11-26 20:23:19.222358235 +0000
--- tests/A-009-002-M-ok.outhyp	2015-11-27 11:48:02.161223905 +0000
***************
*** 8,15 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: [s1] {A Walk in the Desert}
! Somewhere in space
  Menu Principal
  1 - Novo
  2 - Abrir
--- 8,14 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: Somewhere in space
  Menu Principal
  1 - Novo
  2 - Abrir
