*** tests/A-015-002-M-ok.out	2015-11-26 20:23:19.230358235 +0000
--- tests/A-015-002-M-ok.outhyp	2015-11-26 20:24:56.826359694 +0000
***************
*** 21,27 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Identificador da secção: Identificador único: O nome da secção foi alterado.
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
--- 21,27 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Identificador da secção: Identificador da secção: O nome da secção foi alterado.
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
