*** tests/A-010-003-M-ok.out	2015-11-26 20:23:19.222358235 +0000
--- tests/A-010-003-M-ok.outhyp	2015-11-27 11:48:02.609223927 +0000
***************
*** 102,108 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {A Walk in the Desert}
  [] {New Hope}
  [] {Drama in the Clouds}
  Menu de Edição
--- 102,109 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: {Objects in Space}
! [] {A Walk in the Desert}
  [] {New Hope}
  [] {Drama in the Clouds}
  Menu de Edição
