*** tests/A-014-002-M-ok.out	2015-11-24 08:56:06.000000000 +0000
--- tests/A-014-002-M-ok.outhyp	2015-11-25 21:21:44.514956771 +0000
***************
*** 57,63 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {A Walk in the Desert}
  [] {Sec 2}
  [] {Patience You Must Have}
  [] {Drama in the Clouds}
--- 57,64 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: {Objects in Space}
! [] {A Walk in the Desert}
  [] {Sec 2}
  [] {Patience You Must Have}
  [] {Drama in the Clouds}
