*** tests/A-016-003-M-ok.out	2015-11-24 08:56:06.000000000 +0000
--- tests/A-016-003-M-ok.outhyp	2015-11-25 21:21:45.706956759 +0000
***************
*** 34,40 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {A Walk in the Desert}
  [] {Patience You Must Have}
  Menu de Edição
  1 - Mudar título
--- 34,41 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: {Objects in Space}
! [] {A Walk in the Desert}
  [] {Patience You Must Have}
  Menu de Edição
  1 - Mudar título
***************
*** 50,58 ****
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
- [] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
- [] {Patience You Must Have}
  Do or do not.
  There is no try.
  Menu de Edição
--- 51,57 ----
