*** tests/A-016-009-M-ok.out	2015-11-19 19:53:11.000000000 +0000
--- tests/A-016-009-M-ok.outhyp	2015-12-06 23:07:23.888878275 +0000
***************
*** 227,233 ****
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 206
! Identificadores únicos: 1
  Menu Principal
  1 - Novo
  2 - Abrir
--- 227,233 ----
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 206
! Identificadores únicos: 0
  Menu Principal
  1 - Novo
  2 - Abrir
