*** tests/A-018-002-M-ok.out	2015-11-26 20:23:19.234358235 +0000
--- tests/A-018-002-M-ok.outhyp	2015-11-26 20:24:58.250359715 +0000
***************
*** 35,41 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Identificador único: Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
--- 35,42 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Número do parágrafo: O nome do parágrafo foi alterado.
! Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
***************
*** 71,77 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: Where are they?
  Menu Principal
  1 - Novo
  2 - Abrir
--- 72,78 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'P1.2' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 109,115 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Identificador único: O nome do parágrafo foi alterado.
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
--- 110,116 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Número do parágrafo: O nome do parágrafo foi alterado.
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
***************
*** 137,150 ****
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
- [] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
  Where are they?
  Who knows?
- [] {Patience You Must Have}
  Do or do not.
  There is no try.
- [] {Drama in the Clouds}
  Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
--- 138,148 ----
***************
*** 171,177 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: Where are they?
  Menu Principal
  1 - Novo
  2 - Abrir
--- 169,175 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'NP1.2' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 197,203 ****
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 3
  Dimensão do documento (bytes): 294
! Identificadores únicos: 1
  Menu Principal
  1 - Novo
  2 - Abrir
--- 195,201 ----
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 3
  Dimensão do documento (bytes): 294
! Identificadores únicos: 0
  Menu Principal
  1 - Novo
  2 - Abrir
