*** tests/A-014-003-M-ok.out	2015-11-26 20:23:19.230358235 +0000
--- tests/A-014-003-M-ok.outhyp	2015-11-26 20:24:56.558359690 +0000
***************
*** 45,51 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Sub 1.1}
  [] {Sub 1.2}
  Menu de Edição
  1 - Mudar título
--- 45,52 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Patience You Must Have}
! [] {Sub 1.1}
  [] {Sub 1.2}
  Menu de Edição
  1 - Mudar título
***************
*** 73,79 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Sub 1.1}
  [] {Sub 1.2}
  [] {Sec 1,3}
  Menu de Edição
--- 74,81 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Patience You Must Have}
! [] {Sub 1.1}
  [] {Sub 1.2}
  [] {Sec 1,3}
  Menu de Edição
***************
*** 89,95 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Sub 1.1}
  [] {Sub 1.2}
  [] {Sec 1,3}
  Menu de Edição
--- 91,98 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Patience You Must Have}
! [] {Sub 1.1}
  [] {Sub 1.2}
  [] {Sec 1,3}
  Menu de Edição
