*** tests/A-018-003-M-ok.out	2015-11-26 20:23:19.234358235 +0000
--- tests/A-018-003-M-ok.outhyp	2015-11-27 11:48:06.705224122 +0000
***************
*** 71,77 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: These aren’t the droids you’re looking for.
  Menu Principal
  1 - Novo
  2 - Abrir
--- 71,77 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'P1' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 145,151 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: NOOOOOOOOOOOOOOOOOOO...
  Menu Principal
  1 - Novo
  2 - Abrir
--- 145,151 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'P1' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 160,166 ****
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 3
  Dimensão do documento (bytes): 294
! Identificadores únicos: 3
  Menu Principal
  1 - Novo
  2 - Abrir
--- 160,166 ----
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 3
  Dimensão do documento (bytes): 294
! Identificadores únicos: 2
  Menu Principal
  1 - Novo
  2 - Abrir
