*** tests/A-016-008-M-ok.out	2015-11-19 19:43:06.000000000 +0000
--- tests/A-016-008-M-ok.outhyp	2015-12-06 23:07:23.720878278 +0000
***************
*** 156,164 ****
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
! [sec1] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
! [sec2] {Drama in the Clouds}
  Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
--- 156,164 ----
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
! [] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
! [] {Drama in the Clouds}
  Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
***************
*** 189,195 ****
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 218
! Identificadores únicos: 2
  Menu Principal
  1 - Novo
  2 - Abrir
--- 189,195 ----
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 218
! Identificadores únicos: 0
  Menu Principal
  1 - Novo
  2 - Abrir
