*** tests/A-016-008-M-ok.out	2015-11-19 19:43:06.000000000 +0000
--- tests/A-016-008-M-ok.outhyp	2015-12-07 10:59:59.996011248 +0000
***************
*** 189,195 ****
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 218
! Identificadores únicos: 2
  Menu Principal
  1 - Novo
  2 - Abrir
--- 189,195 ----
  Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
  Secções de topo: 2
  Dimensão do documento (bytes): 218
! Identificadores únicos: 3
  Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 200,206 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'Sec1.1' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
--- 200,206 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: [Sec1.1] {Sec 1-1}
  Menu Principal
  1 - Novo
  2 - Abrir
