*** tests/A-019-002-M-ok.out	2015-11-26 20:23:19.234358235 +0000
--- tests/A-019-002-M-ok.outhyp	2015-11-27 11:48:07.041224138 +0000
***************
*** 100,106 ****
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: New content
  Menu Principal
  1 - Novo
  2 - Abrir
--- 100,106 ----
  7 - Mostrar elemento de texto
  8 - Editar
  0 - Sair
! Escolha uma opção: Identificador do elemento de texto: O elemento de texto 'P1' não existe.
  Menu Principal
  1 - Novo
  2 - Abrir
