*** tests/A-011-002-M-ok.out	2015-11-26 20:23:19.222358235 +0000
--- tests/A-011-002-M-ok.outhyp	2015-11-26 20:24:55.394359673 +0000
***************
*** 45,51 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Sub 1.1}
  [] {Sub 1.2}
  Menu de Edição
  1 - Mudar título
--- 45,52 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Patience You Must Have}
! [] {Sub 1.1}
  [] {Sub 1.2}
  Menu de Edição
  1 - Mudar título
***************
*** 87,93 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Sub 2.1}
  [] {Sub 2.2}
  Menu de Edição
  1 - Mudar título
--- 88,95 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Drama in the Clouds}
! [] {Sub 2.1}
  [] {Sub 2.2}
  Menu de Edição
  1 - Mudar título
