*** tests/A-010-004-M-ok.out	2015-11-26 20:23:19.222358235 +0000
--- tests/A-010-004-M-ok.outhyp	2015-11-26 20:24:55.094359668 +0000
***************
*** 102,108 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Where is Leia?}
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
--- 102,109 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {Drama in the Clouds}
! [] {Where is Leia?}
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
