*** tests/A-017-003-M-ok.out	2015-11-26 20:23:19.230358235 +0000
--- tests/A-017-003-M-ok.outhyp	2015-11-26 20:24:57.974359711 +0000
***************
*** 79,91 ****
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
- [] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
- [] {Patience You Must Have}
  Do or do not.
  There is no try.
  P3
- [] {Drama in the Clouds}
  Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
--- 79,88 ----
