*** tests/A-018-004-M-ok.out	2015-11-18 16:49:49.000000000 +0000
--- tests/A-018-004-M-ok.outhyp	2015-12-06 23:07:25.352878251 +0000
***************
*** 62,68 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
  Where are they?
  Who knows?
--- 62,68 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [sec1] {A Walk in the Desert}
  These aren’t the droids you’re looking for.
  Where are they?
  Who knows?
