*** tests/A-016-002-M-ok.out	2015-11-26 20:23:19.230358235 +0000
--- tests/A-016-002-M-ok.outhyp	2015-11-27 11:48:05.357224058 +0000
***************
*** 34,40 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: [] {A Walk in the Desert}
  [] {Drama in the Clouds}
  Menu de Edição
  1 - Mudar título
--- 34,41 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: {Objects in Space}
! [] {A Walk in the Desert}
  [] {Drama in the Clouds}
  Menu de Edição
  1 - Mudar título
***************
*** 50,57 ****
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: {Objects in Space}
- [] {A Walk in the Desert}
- [] {Drama in the Clouds}
  Menu de Edição
  1 - Mudar título
  2 - Listar secções
--- 51,56 ----
