*** tests/A-018-007B-M-ok.out	2015-12-02 18:54:07.000000000 +0000
--- tests/A-018-007B-M-ok.outhyp	2015-12-07 11:00:02.004011246 +0000
***************
*** 48,54 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Identificador único: Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
--- 48,55 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Identificador único: O nome do parágrafo foi alterado.
! Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
