*** tests/A-020-001-M-ok.out	2015-11-26 20:23:19.234358235 +0000
--- tests/A-020-001-M-ok.outhyp	2015-11-27 18:44:58.278263969 +0000
***************
*** 62,67 ****
--- 62,68 ----
  11 - Remover parágrafo
  0 - Sair
  Escolha uma opção: [] {Drama in the Clouds}
+ Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
  Menu de Edição
***************
*** 97,102 ****
--- 98,104 ----
  Do or do not.
  There is no try.
  [] {Drama in the Clouds}
+ Obi-Wan never told you what happened to your father.
  He told me enough! He told me YOU killed him.
  NOOOOOOOOOOOOOOOOOOO...
  Menu de Edição
