*** tests/A-020-002-M-ok.out	2015-11-24 08:56:06.000000000 +0000
--- tests/A-020-002-M-ok.outhyp	2015-11-25 21:21:47.382956741 +0000
***************
*** 52,107 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Menu de Edição
! 1 - Mudar título
! 2 - Listar secções
! 3 - Mostrar conteúdo
! 4 - Seleccionar secção
! 5 - Inserir secção
! 6 - Nomear secção
! 7 - Remover secção
! 8 - Inserir parágrafo
! 9 - Nomear parágrafo
! 10 - Alterar parágrafo
! 11 - Remover parágrafo
! 0 - Sair
! Escolha uma opção: [] {Drama in the Clouds}
! Obi-Wan never told you what happened to your father.
! NOOOOOOOOOOOOOOOOOOO...
! Menu de Edição
! 1 - Mudar título
! 2 - Listar secções
! 3 - Mostrar conteúdo
! 4 - Seleccionar secção
! 5 - Inserir secção
! 6 - Nomear secção
! 7 - Remover secção
! 8 - Inserir parágrafo
! 9 - Nomear parágrafo
! 10 - Alterar parágrafo
! 11 - Remover parágrafo
! 0 - Sair
! Escolha uma opção: Menu de Edição
! 1 - Mudar título
! 2 - Listar secções
! 3 - Mostrar conteúdo
! 4 - Seleccionar secção
! 5 - Inserir secção
! 6 - Nomear secção
! 7 - Remover secção
! 8 - Inserir parágrafo
! 9 - Nomear parágrafo
! 10 - Alterar parágrafo
! 11 - Remover parágrafo
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Novo
! 2 - Abrir
! 3 - Guardar
! 4 - Listar meta-informação
! 5 - Adicionar autor
! 6 - Listar secções
! 7 - Mostrar elemento de texto
! 8 - Editar
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 52,55 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: 
\ No newline at end of file
