*** tests/A-020-004-M-ok.out	2015-11-26 20:23:19.234358235 +0000
--- tests/A-020-004-M-ok.outhyp	2015-11-27 11:48:07.481224159 +0000
***************
*** 21,28 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: O parágrafo '0' não existe.
! Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
--- 21,27 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
***************
*** 49,56 ****
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: O parágrafo '2' não existe.
! Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
--- 48,54 ----
  10 - Alterar parágrafo
  11 - Remover parágrafo
  0 - Sair
! Escolha uma opção: Número do parágrafo: Menu de Edição
  1 - Mudar título
  2 - Listar secções
  3 - Mostrar conteúdo
